feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/amountChooser/BaseAmountChooserProvider.kt

```diff
@@ -31,18 +60,115 @@ open class BaseAmountChooserProvider(
             inputState.map { input -> input.parseBigDecimalOrNull() }
         }.share()
 
-    override val amount: Flow<BigDecimal> = amountState
+    private val _amount: Flow<BigDecimal> = amountState
         .map { it.value.orZero() }
         .share()
 
+    @Deprecated("Use amountState instead")
+    override val amount: Flow<BigDecimal> = _amount
```

Comment:
Why do we use both variables if we can have only one?

Reply:
To avoid deprecation warnings inside BaseAmountChooserProvider itself